internal/credential/vault/repository_credential_store.go

```diff
@@ -690,6 +690,15 @@ func (r *Repository) UpdateCredentialStore(ctx context.Context, cs *CredentialSt
 			}
 			msgs = append(msgs, &csOplogMsg)
 
+			ir, err := NewRepository(reader, w, r.kms)
```

Comment:
Since you are doing a lookup anyway is there a reason we don't do the lookup at the end of this method and return that with all necessarily fields blanked out

Reply:
Changed.  Thanks!